examples/dapperish.go

```diff
@@ -36,8 +35,8 @@ func client() {
 
 		httpClient := &http.Client{}
 		httpReq, _ := http.NewRequest("POST", "http://localhost:8080/", bytes.NewReader([]byte(text)))
-		opentracing.AddTraceContextToHeader(
-			span.TraceContext(), httpReq.Header, opentracing.GlobalTracer())
+		opentracing.PropagateSpanInHeader(
```

Comment:
Existing tracers send via multiple headers, but I guess this is named PropagateSpanInHeader vs PropagateSpanInHeaders bc in go, headers are in an object named Header?